src/componentes/unmounted/Auth/AllowProfile.test.tsx

```
@@ -0,0 +1,49 @@
+// Elementos de teste para error e children
+const DummyError = () => <div data-testid="error">Erro: acesso negado</div>
+const DummyChild = () => <div data-testid="child">Conteúdo permitido</div>
+
+const user = {
+  id: "e9b07b5c-4e51-4324-a5e2-93f3cf09a330",
+  nome: "Usuário Teste",
+  mail: "usuario@mail.com",
+  cargo: "Cargo Teste",
+  municipio: "Impulsolandia",
+  equipe: "Equipe Teste",
+  municipio_id_sus: "111111",
```

Biome (1.9.4)

[error] This object property name should be in camelCase.



(lint/style/useNamingConvention)